src/main.ts

```diff
@@ -43,11 +45,39 @@ const command: Command = {
 				child.on('exit', (code) => (code !== 0 ? reject(new Error(errorMessage)) : resolve()));
 			});
 
+		let tmpDir: string;
+		const rmTmpDir = () => tmpDir && (fs.unlinkSync(join(tmpDir, 'tsconfig.json')), fs.rmdirSync(tmpDir));
 		const spinner = ora(`building ${name} theme`).start();
 		return createTask((callback: any) => rimraf(join('dist', 'src', name), callback))
 			.then(() => createChildProcess('tcm', [join('src', name, '*.m.css')], 'Failed to build CSS modules'))
 			.then(() =>
-				createChildProcess('tsc', ['--outDir', join('dist', 'src', name)], `Failed to build ${name}/index.d.ts`)
+				createTask((callback: any) =>
+					fs.mkdtemp(os.tmpdir(), (error: Error | undefined, folder: string) => {
+						if (error) {
+							callback(error);
+						} else {
+							tmpDir = folder;
+							fs.writeFile(
```

Comment:
Hmm, the prettier output here is not really great. Sorry to be a pain, but this might be easier to read if the paths and content were split into their variables and then passed to `fs.writeFile`.